types/react-easy-crop/tslint.json

```diff
@@ -0,0 +1,80 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Is there a good reason to disable all of these rules? Since this is a new definition, and is relatively small, it would be better to use the default ruleset and follow the style guidelines of the DefinitelyTyped repository.

Reply:
Good points.

I've removed all the disabled rules and made any changes that were required following this.